neurom/view/dendrogram.py

```diff
@@ -54,13 +55,13 @@ def __init__(self, neurom_section):
             if isinstance(neurom_section, Neurite):
                 neurom_section = neurom_section.root_node
             segments = neurom_section.points
-            segment_lengths = np.linalg.norm(np.diff(segments[:, COLS.XYZ], axis=0), axis=1)
+            lengths = interval_lengths(segments)
```

Comment:
The optimization is great but please return back the original name `segment_lengths`. It is necessary in the context of other names: `segments` and `segment_radii`.

Reply:
Actually, I prefer to not use the word `segments` to store `neurom_section.points` so I removed the variable.

Reply:
Well, there were 3 vars in a row: `segments`, `segment_lengths` and `segment_radii`. Now because of your renaming there are: `segments`, `lengths` and `segment_radii`. In order to be coherent in naming it's either should stay as before or create new names for `segments` and `segment_radii` to better reflect their meaning.

Reply:
Yes. I changed it in the last patch.